### ISSUE TITLE
[0.9.x] meter_power capabilty unable to handle negative value

### ISSUE THREAD

**Issue by gruijter** [athombv/homey#775]
**What is your Homey version (Settings → System)?**

0.9.2

**What did you try to achieve?**
in Smile P1 app the standard capability meter_power is used. Since firmware 0.9 this capability does not work anymore with negative values

*Please include detailed steps how we can reproduce the issue*

use a negative value for meter_power

**What was the result?**

The device card gives an error "meter_power:out_of_range"
the meter_power is also not logged in insights

**What did you expect as result?**

negative vakues to work (like it did before fw 0.9)

**Any other remarks**

see also forum discussion and screenshot on https://forum.athom.com/discussion/comment/28921#Comment_28921

**Comment by gruijter** [comment]
I found that the problem is caused by recently introduced "min" value of the default meter_power capability:
```"meter_power": {
			"type": "number",
			"title": {
				"en": "Power Meter",
				"nl": "Stroommeter"
			},
			"min": 0,
			"decimals": 2,
			"units": {
				"en": "Wh"
			},
			"desc": {
				"en": "Power usage in Watts per Hour (Wh)",
				"nl": "Stroomverbuik in Watt per uur (Wh)"
			},
			"getable": true,
			"setable": false```

I also found that I can override the default value of 0 to a large negative number. This however only works right after pairing. If the app resets (e.g. stop/start, reboot) the min value defaults again to 0 making the app crash.

**Comment by bvdbos** [comment]
With resetting Homey the app keeps working for me. Stopping and restarting the app triggers the issue you mention. Rebooting Homey fixes this...

**Comment by WeeJeWel** [comment]
Fixed in 0.9.3. Also changed units to kWh.
